### PR TITLE
add: `xapps-common-deb`

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

File: packages/xapps-common-deb/xapps-common-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name="xapps-common-deb"
+gives="xapps-common"
+version="2.4.1"
+url="http://ftp.debian.org/debian/pool/main/x/xapp/xapps-common_2.4.1-1_all.deb"
+description="Xapp provides components which are common to multiple desktop environments and required to implement cross-DE solutions"
```
Comment (Henryws) [inline packages/xapps-common-deb/xapps-common-deb.pacscript:5]: No package name in description.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/xapps-common-deb/xapps-common-deb.pacscript (hunk 1 above)
Comment (Zahrun) [reply packages/xapps-common-deb/xapps-common-deb.pacscript]: Ok

**Review by Henryws — CHANGES_REQUESTED** [review]
Add `hash`.

**Review by Henryws — CHANGES_REQUESTED** [review]
Add `repology` array.

**Comment by Zahrun** [comment]
Ok

**Review by Henryws — APPROVED** [review]

**Pull request merged**